pkg/restapi/vc/operation/operations.go

```diff
@@ -111,11 +181,61 @@ func createCredential(data *CreateCrendential) (*verifiable.Credential, error) {
 	return validatedCred, nil
 }
 
+func (c *Operation) createProfile(pr *ProfileRequest) (*ProfileResponse, error) {
+	if pr.DID == "" || pr.URI == "" {
+		return nil, fmt.Errorf("profile request is bad")
```

Comment:
error should be more informative: missing DID and/or URI information

Reply:
done